website/docs/general/2fa.md

```diff
@@ -5,26 +5,26 @@ keywords:
 - API gateway
 - APISIX
 - Apache APISIX
-- two factor authentication
-description: This article provides information of how to enable Two-factor authentication(2FA) on GitHub. It consists of 3 parts, what is Two-factor authentication(2FA), how to enable 2FA on GitHub, and how to submit codes.
+- Two Factor Authentication
+description: This article provides information of how to enable Two-Factor Authentication (2FA) on GitHub. It consists of 3 parts, what is Two-Factor Authentication (2FA), how to enable 2FA on GitHub, and how to submit code.
 ---
 
-## Two-factor authentication(2FA)
+## Two-Factor Authentication (2FA)
 
-Two factor authentication (2FA) refers to the authentication method that combines both passport and an object (credit card, SMS phone, token or biomarkers as fingerprint) to identify a user. To ensure the security of the committer’s account, we need you to enable 2FA to sign in and contribute codes on GitHub. More details, please refer to 2FA.
+Two-Factor Authentication (2FA) refers to the authentication method that combines both a passport and an object (credit card, SMS, tokens or biomarkers as fingerprint) to identify a user. 
```

Comment:
```suggestion
Two-Factor Authentication (2FA) refers to the authentication method that combines both a password and an object (credit card, SMS, tokens, or biomarkers as fingerprint) to identify a user. 
```
I should be **password**, right?

Reply:
Yes I think so too. I did not change it thinking that it might be a thing. Will make the change.